simulator/src/window.rs

```diff
@@ -87,10 +90,19 @@ impl Window {
                 } => {
                     return true;
                 }
+                Event::MouseButtonUp { x, y, .. } => {
+                    self.input_events
+                        .push(Point::new(x / self.scale as i32, y / self.scale as i32));
```

Comment:
This doesn't take the pixel spacing into account: https://github.com/jamwaffles/embedded-graphics/blob/e3412829cb16f19ec3d5a80a007e4f0b61132a47/simulator/src/window.rs#L69-L72